doc/reference/ovn-internals.md

```diff
@@ -0,0 +1,168 @@
+# LXD OVN implementation
```

Comment:
I think `(ovn-internals)=` needs to come before the heading for doc builds. Just something I noticed while working on https://github.com/canonical/lxd/pull/14574 :)

Reply:
This is required for external linking but there are no links to this page yet.

Reply:
Ah I see, thanks.